scripts/objects/alienShipManager.js

```diff
@@ -0,0 +1,143 @@
+Game.objects.AlienShipManager = function (spec) {
+    'use strict';
+    let ships = []; 
+
+    function createNewShip(shipSpec) {
```

Comment:
The individual ship could be moved into its own object in a separate file at some point.